src/libraries/System.Private.CoreLib/src/System/SearchValues/IndexOfAnyAsciiSearcher.cs

```diff
@@ -17,18 +17,36 @@ internal static class IndexOfAnyAsciiSearcher
     {
         public struct AsciiState(Vector128<byte> bitmap, BitVector256 lookup)
         {
-            public Vector256<byte> Bitmap = Vector256.Create(bitmap);
+            public Vector512<byte> Bitmap512 = Vector512.Create(bitmap);
```

Comment:
These can't / shouldn't be readonly?

Reply:
The `TryIndexOfAny` helpers (called by non-searchvalues `IndexOfAny`) are writing into this field directly when computing the bitmap to avoid needing an extra local.
As far as the vectorized code is concerned, these could be readonly.